themes/ropensci/layouts/blog/single.html

```diff
@@ -35,8 +35,9 @@ <h2 style="text-align: center;">{{ .Title }}</h2>
                 {{ range .Params.tags }}
                     <a href="{{ "/tags/" | relLangURL }}{{ . | urlize }}"><span class="label">{{ . }}</span></a>
                 {{ end }}
-            </div>
+                <br>
```

Comment:
I reckon it's probably not very professional :woman_shrugging:

Reply:
😸